test/src/main/java/org/apache/accumulo/test/BadDeleteMarkersCreatedIT.java

```diff
@@ -173,6 +173,7 @@ public void test() throws Exception {
       }
       Assert.fail("Delete entry should have been deleted by the garbage collector: " + entry.getKey().getRow().toString());
     }
+    scanner.close();
```

Comment:
This could also be try-with-resources. There isn't an existing try block, but one could be added. Technically, without one, it's a bit unsafe anyway, because if it's not there, then we probably should put the close call in a finally block.

I think the only time where we can't use try-with-resources syntax is either:
1. we are intentionally testing what happens when the scanner is not closed, or
2. we construct the scanner in one method, and pass it to another, where it is closed later.

Reply:
Okay that makes sense, I will do another pass through trying to do this.  and I will avoid those 2 exceptions.